lib/modelHelpers.py

```diff
@@ -136,23 +116,31 @@ def make_prompt_from_editor_ctx(ctx: str):
 
 
 def format_message(content: str):
```

Comment:
Good for right now in this PR; I think we may eventually want to refactor this to use [Python protocols](https://peps.python.org/pep-0544/) to enforce a static structurally typed interface. But nothing wrong with this if it works for the time being. 

I can do that python class refactoring and experiment a bit since I know that it is very Python-specific. Will hold off any changes to this branch so I don't disrupt this PR; learned my lesson with the last PR and want to make things small and atomic.